Automattic/sensei
# Remove grade avg and course column from Reports->Lessons

Relates to [#4835](https://github.com/Automattic/sensei/issues/4835)

### Changes proposed in this Pull Request

Removed 'Average Grade' and 'Course' column from Reports->Lessons table

### Testing instructions

1.  Add a few lessons
2. Check Reports->Lessons page and see if 'Average Grade' and 'Course' is removed
 
### Screenshot / Video

![image](https://user-images.githubusercontent.com/6820724/156555649-7880542a-149b-4e31-88ae-c4bc679f9a06.png)
